src/client/src/apps/SpotlightRoute/Spotlight.tsx

```diff
@@ -39,8 +43,15 @@ const Contacting = styled.span`
 
 const INPUT_ID = 'spotlight'
 
+const autobahn = new AutobahnConnectionProxy(
+  process.env.REACT_APP_BROKER_HOST || location.hostname,
+  'com.weareadaptive.reactivetrader',
+  +(process.env.REACT_APP_BROKER_PORT || location.port),
```

Comment:
very minor and edge case: looks like port is a number, which means that if `process.env.REACT_APP_BROKER_PORT` is set is 0, it will be disregarded in favor of `location.port`

Reply:
I'm not sure 0 would be valid anyway..